src/ring_theory/ideal/basic.lean

```diff
@@ -420,6 +420,38 @@ lemma bot_is_maximal {K : Type u} [field K] : is_maximal (⊥ : ideal K) :=
 ⟨λ h, absurd ((eq_top_iff_one (⊤ : ideal K)).mp rfl) (by rw ← h; simp),
 λ I hI, or_iff_not_imp_left.mp (eq_bot_or_top I) (ne_of_gt hI)⟩
 
+/-- Maximal ideals in a non-field are nontrivial. -/
+variables {R : Type u} [comm_ring R]
+lemma maximal_gt_bot {M : ideal R} (hm : M.is_maximal) (non_field : ∃(I : ideal R), ⊥ < I ∧ I < ⊤) : ⊥ < M :=
```

Comment:
This should be called `bot_lt_of_maximal`.
Also, for better or for worse, `is_maximal` is currently a typeclass, so `M` should get `()`-brackets, and `hm` should get `[]`.